### PR TITLE
Add streaming e2e tests

### PR DESCRIPTION
**Pull request by objectiser**
Signed-off-by: Gary Brown <gary@brownuk.com>

### CODE REVIEW

File: test/kafka.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+apiVersion: kafka.strimzi.io/v1alpha1
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  kafka:
+    version: 2.1.0
+    replicas: 1
+    listeners:
+      plain: {}
```
Comment (jpkrohling) [inline test/kafka.yml]: Are those empty nodes required?

File: test/kafka.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      tls: {}
+    config:
+      offsets.topic.replication.factor: 1
+      transaction.state.log.replication.factor: 1
+      transaction.state.log.min.isr: 1
+      log.message.format.version: "2.1"
+    storage:
+      type: persistent-claim
+      size: 100Gi
```
Comment (jpkrohling) [inline test/kafka.yml]: O_O
I hope it won't create a 100Gi volume?

File: test/kafka.yml (hunk 1 above)
Comment (objectiser) [reply test/kafka.yml]: Theses are a straight copy from the strimzi installation instructions - aiming to keep things straightforward, so periodically we can update to more recent versions.

File: test/kafka.yml (hunk 2 above)
Comment (objectiser) [reply test/kafka.yml]: See comment above.

Comment (jpkrohling) [reply test/kafka.yml]: OK, I was just afraid it would create a real 100Gi volume :-) If it doesn't, it's all good then

**Pull request merged**